switchlink/switchlink_route.c

```diff
@@ -246,6 +247,11 @@ void switchlink_process_route_msg(const struct nlmsghdr *nlmsg, int msgtype) {
     if (oif_valid) {
       switchlink_db_status_t status;
       status = switchlink_db_get_interface_info(oif, &ifinfo);
+      if(ifinfo.link_type == SWITCHLINK_LINK_TYPE_BOND) {
```

Comment:
first check status, if successful then access the ifinfo structure.

Reply:
Added a safe check here.